package.json

```diff
@@ -13,12 +13,14 @@
     "cache:clear": "rm -rf .cache",
     "evals": "npm run build-dom-scripts && tsx evals/index.eval.ts",
     "build-dom-scripts": "tsx lib/dom/genDomScripts.ts",
-    "build": "npm run build-dom-scripts && tsup lib/index.ts --dts",
-    "release": "changeset publish"
+    "build-types": "tsc --emitDeclarationOnly --outDir dist",
+    "build-js": "esbuild lib/index.ts --bundle --platform=node --outdir=dist --format=esm --minify --tree-shaking=true --packages=external",
```

Comment:
Would love a second pair of eyes here to make sure I did this right

Reply:
one thing that sticks out is i think this package is commonjs but the output format is esm; i'm not sure if we actually use anything commonjs specific, or if this package is commonjs for a reason, but we should be thorough